doc/decisions/STEPS.md

```diff
@@ -0,0 +1,144 @@
+3. After a decision is merged as "Decided", it can be moved to "Partially Implemented" or "Implemented" by any PR.
+
+The rest of the document describes the steps in full details.
+
+## Drafts
+
+> This step is highly recommended and it is even required if the problem is not yet clear to all the core developers.
+
+The first step is to create a PR with:
+
+- **one** decision, where at least the "Problem" is filled out and "Decision", "Rationale" and "Implications" are **not** yet filled out.
+- optional backlinks from related decisions.
+
+This step is brainstorming for completely new ideas.
+
+> No agreement about anything is needed to merge decisions in this stage.
+
+## Problem Clear
+
+This step is very important:
+
+- it clarifies the importance of the problem, answering:
+  Why should we put our time and energy in this problem and not in another problem.
+- it clarifies the scope of the decision.
+- it clarifies relation to other problems.
+
+Decisions will have much smoother further steps if this step is done carefully without prejudice.
+It is especially important that one shouldn't have a fixed mind-set about a preferred solution from the beginning.
+
+> Everyone must agree that the problem exists and is worth solving so that a decision PR in "Problem Clear" step can be merged.
+> A problem is clear if everyone would be able to describe an experiment or test case that shows if a solution fixes the problem.
+
+## Solutions Clear
+
+> This step is recommended if it is not yet clear to the core developers which solution is the best.
+
+Here you must ensure:
+
+- problem, constraint and assumptions are well-explained and sound
+- links from/to related decisions are created
+- there are several solutions described, each with rationale and implication
+- "Decision", "Rationale" and "Implications" are **not** yet filled out if there are people arguing for different options
+
+Here the decision should not only have one decision but should describe several solutions.
+For each solution a proposal, rationale and optionally implications should be given.
+
+> The solutions are clear if all reviewers understand the given solutions and no reviewer can come up with better solutions.
+> The solution space is clear.
+> I.e. the trade-offs, combinations and pros/cons of the solutions are explored.
+> Decision author and reviewers are satisfied that every useful solution is present in the decision.
+
+## In Review
+
+Now it is allowed to have the decision from the previous round in the "Decision" section.
+In this step, the last details of the chosen decision get polished:
+
+- consistency with other decisions
+- links from/to related decisions checked
+- "Decision", "Rationale" and "Implications" are fully filled out
+
+> Without merges in between, this is the last step reachable for a decision PR that started in "Drafts".
```

Comment:
IMO creating a PR that introduces a new decision file and adds it directly to `doc/decisions/X_decided` should be fine. Since we need the reviews anyway, such a PR only makes sense, if the decision is only documenting status quo, consequences of other decisions, etc. Basically only decisions were there is only one realistic option.

Reply:
If the decisions are only documentation is does not really matter.